docs/source/howto/codes.rst

```diff
@@ -45,8 +45,90 @@ Design guidelines
 Running external codes
 ======================
 
-`#3987`_
+To run an external code with AiiDA, you will need to use an appropriate :ref:`calculation plugin <topics:plugins>` that knows how to transform the input nodes into the input files that the code expects, copy everything in the code's machine, run the calculation and retrieve the results.
+This plugin would have already been created, either by you (see section on :ref:`interfacing external codes <how-to:codes:plugin>`) or by other AiiDA plugin-developers (available packages are listed in the `plugin registry <https://aiidateam.github.io/aiida-registry/>`_), and has been properly installed in your environment.
+You can check which ones you have currently available:
 
+.. code-block:: bash
+
+    $ verdi plugin list aiida.calculations
+
+As an example, we will show how to use the ``arithmetic.add`` plugin, which is a pre-installed plugin implemented in `bash` to sum two integers.
+You can access it with the ``CalculationFactory``:
+
+.. code-block:: python
+
+    from aiida.plugins import CalculationFactory
+    calculation_class = CalculationFactory('arithmetic.add')
```

Comment:
Just making sure, this is actually a class, right? And not an instance of a class, in which case it should be:
```suggestion
    calculation = CalculationFactory('arithmetic.add')
```

Reply:
Tis a class. Proof:
```
In [1]: from aiida.plugins import CalculationFactory
In [2]: import inspect
In [3]: calculation_class = CalculationFactory('arithmetic.add') 
In [4]: inspect.isclass(calculation_class)                                                                              
Out[4]: True
```